pages/createUpdate.js

```diff
@@ -110,7 +110,7 @@ class CreateUpdatePage extends React.Component {
                 </Container>
               </Link>
             </BackButtonWrapper>
-            <Container width={1} maxWidth={650}>
+            <Container width={1} maxWidth={680}>
```

Comment:
@SudharakaP Was this part of a design update? Is it required for video embedding?

Reply:
Addition of the video button on the rich text editor toolbar make it like this on the updates page; 😉 

![image](https://user-images.githubusercontent.com/12435965/141541101-cb70aadd-340f-487f-9f35-4d7368a543fc.png)

So we need to change the width a bit.

Reply:
Ok, this might have an impact on the responsiveness but let's not block on this

Reply:
I tested this on various screen sizes and I don't see an apparent loss of responsiveness; 

![Screen Shot 2021-11-17 at 11 28 32](https://user-images.githubusercontent.com/12435965/142269369-a79295ec-ee3b-4cdf-a5cf-6202ce447286.png)

If you see any issues let me know I'll open a new issue to fix that. 😉